### DISCUSSION TITLE
Add restriction on `:bool` fields in schema for possible values

### DISCUSSION THREAD

**Discussion by notramo** [martenframework/marten#204] [Ideas]
Sometimes a checkbox must be checked (ToS, privacy policy, etc.) in order to successfully submit a form. The `:bool` schema field could utilize a parameter for this to enforce only `true` or only `false` values.

**Comment by ellmetha** [comment]
Sorry for the late answer! Why not use a `bool` field that is required?

For example:

```crystal
class MySchema < Marten::Schema
  field :tos, :bool, required: true
end
```

Validation won't pass unless the value is true. It is worth noting that `bool` fields don't differentiate between no values and falsey values at the moment. 

If you need to enforce only `false` values, then I'd argue that the field should probably be refactored (renamed) so that you enforce a `true` value. This can also be achieved with custom code validation rules obviously.
